sass/components/_description_list.scss

```diff
@@ -7,3 +7,9 @@
   @extend .ecl-u-bg-blue-5;
   padding: $ecl-spacing-s;
 }
+
+.ecl-description-list--horizontal {
```

Comment:
Not sure if this is the final solution here. We had the same issue on the horizontal description list paragraph and what we did there is we added the ecl-editor tag to the contents of the description. Also, what happens if the contents of the description are not inside a p? Would we have the issue then?

Reply:
Yep, the right way of doing this is via the ecl-editor class, as the screenshot below shows. I'm going to take care of it.

![image](https://user-images.githubusercontent.com/153362/112833077-7c4f5480-9096-11eb-80fd-7f2ab07719ab.png)